src/languages/fr-FR.js

```diff
@@ -0,0 +1,119 @@
+const { Language, util } = require('klasa');
+
+module.exports = class extends Language {
+
+	constructor(...args) {
+		super(...args);
+		this.language = {
+			DEFAULT: (key) => `${key} n'a pas encore été traduit en 'fr-FR'.`,
+			SETTING_GATEWAY_EXPECTS_GUILD: 'Le paramètre <Guild> attend soit un id soit une instance de Guild.',
+			SETTING_GATEWAY_VALUE_FOR_KEY_NOEXT: (data, key) => `La valeur '${data}' pour la clef '${key}' n'existe pas.`,
+			SETTING_GATEWAY_VALUE_FOR_KEY_ALREXT: (data, key) => `La valeur '${data}' pour la clef '${key}' existe déjà.`,
+			SETTING_GATEWAY_SPECIFY_VALUE: 'Vous devez spécifier une clef pour ajouter ou filtrer.',
+			SETTING_GATEWAY_KEY_NOT_ARRAY: (key) => `La clef '${key}' n'est pas un Array.`,
+			SETTING_GATEWAY_KEY_NOEXT: (key) => `La clef '${key}' n'existe pas dans le schema de data actuel.`,
+			SETTING_GATEWAY_INVALID_TYPE: 'Le paramètre \'type\' doit être soit \'add\' ou \'remove\'.',
+			SETTING_RESOLVER_INVALID_USER: 'Cette clef attend un id ou une instance de User.',
+			SETTING_RESOLVER_INVALID_CHANNEL: 'Cette clef attend un id ou une instance de Channel.',
+			SETTING_RESOLVER_INVALID_GUILD: 'Cette clef attend un id de Guild.',
+			SETTING_RESOLVER_INVALID_ROLE: 'Cette clef attend un id ou une instance de Role.',
+			SETTING_RESOLVER_INVALID_BOOLEAN: 'Cette clef attend un Boolean.',
+			SETTING_RESOLVER_STRING_MAXMIN: (err) => `La longueur de la chaîne doit être de ${err} caractères.`,
+			SETTING_RESOLVER_INVALID_INTEGER: 'Cette clef attend un Integer.',
+			SETTING_RESOLVER_INTEGER_MAXMIN: (err) => `L'entier doit être ${err}.`,
+			SETTING_RESOLVER_INVALID_FLOAT: 'Cette clef attend un Float.',
+			SETTING_RESOLVER_FLOAT_MAXMIN: (err) => `Le float doit être ${err}.`,
+			SETTING_RESOLVER_INVALID_URL: 'Cette clef attend une URL (Uniform Resource Locator).',
+			SETTING_RESOLVER_INVALID_COMMAND: 'Cette clef attend une Command.',
+			SETTING_RESOLVER_MINMAX_EXACTLY: (min) => `exactement ${min}`,
+			SETTING_RESOLVER_MINMAX_BOTH: (min, max) => `entre ${min} et ${max}`,
+			SETTING_RESOLVER_MINMAX_MIN: (min) => `plus que ${min}`,
+			SETTING_RESOLVER_MINMAX_MAX: (max) => `moins que ${max}`,
+			ARG_RESOLVER_INVALID_PIECE: (name, piece) => `${name} doit être un nom de ${piece} valide.`,
```

Comment:
All ARG_RESOLVER strings were updated to just be RESOLVER, as they now serve both ARG_RESOLVER and SETTING_RESOLVER